app/index.js

```diff
@@ -88,6 +91,7 @@ module.exports = class extends Generator {
 
     promise.then(props => {
       this.appDescription = props.appDescription
+      this.keywords = props.keywords.trim().split(',')
```

Comment:
better use space for separation

Reply:
Also keep in mind what happens if the user don't enter nothing

Reply:
Are you sure you want spaces? I've seen may projects with keywords like `functional programming` and they always end up split. 😕

Reply:
I'm not sure 🤔 https://github.com/npm/npm/issues/7072